Makefile

```diff
@@ -19,7 +19,7 @@ OPEN_PR="true"
 # Force the installation of a Concrete Python version, which is very useful with nightly versions
 # /!\ WARNING /!\: This version should NEVER be a wildcard as it might create some
 # issues when trying to run it in the future.
-CONCRETE_PYTHON_VERSION="concrete-python==2.5.0rc1"
+CONCRETE_PYTHON_VERSION="concrete-python==2023.12.5"
```

Comment:
Maybe it's worth to directly switch to 2023.12.6 which contains the truncate?

Reply:
It wasn't available at the moment of this PR, it can be done but this takes some time between updating licences, squashing and the CI

Reply:
Got it let's do it in the truncate PR then thanks!